dist/src/main/java/io/camunda/application/commons/service/CamundaServicesConfiguration.java

```diff
@@ -316,4 +323,20 @@ public AuthorizationChecker authorizationChecker(
       final AuthorizationSearchClient authorizationSearchClient) {
     return new AuthorizationChecker(authorizationSearchClient);
   }
+
+  @Bean
+  public ProcessCache processCache(
+      final GatewayRestConfiguration configuration,
```

Comment:
We shouldn't need a gateway REST configuration for those properties. They are not defined for REST, they are defined for services.

Reply:
https://github.com/camunda/camunda/issues/34945